math/pkg.generated.mbti

```diff
@@ -41,6 +41,9 @@ fn cbrt(Double) -> Double
 
 fn cbrtf(Float) -> Float
 
+#as_free_fn
```

Comment:
It seems that the `#as_free_fn` attribute from the target of `using` is not properly cleaned. Since this is not critical, let's fix this in the next release. cc @bobzhang